ee/tabby-webserver/src/service/email.rs

```diff
@@ -68,6 +91,7 @@ impl EmailService for EmailServiceImpl {
             smtp_server.clone(),
         )
         .await?;
+        *self.from.write().await = smtp_username.clone();
```

Comment:
```suggestion
        // TODO: make from address being configurable in EmailSettings.
        *self.from.write().await = smtp_username.clone();
```

seems to me this should be something configurable in `EmailSettings`. But should be fine to do this in a followup.

Reply:
Would it ever not be the same as the `smtp_username`? I was thinking what people might want to configure is the display name (currently always set to Tabby)

Reply:
I guess it'll likely being something like `noreply@abc.com`, and smtp_username could have access to multiple email address (e.g aws ses)